### ISSUE TITLE
function onCanMoveTo can't be override

### ISSUE THREAD

**Issue by ChristopheCaron** [mbraak/jqTree#791]
Hi,

I followed the documentation and used:
```
 $('#tree1').tree({
    data: data,
    dragAndDrop: true,
    saveState: true,
    onCanMoveTo: function(movedNode, targetNode, position) {
        console.log('test');
        return true;
  }
});

```

but the console.log was never displayed

I used tree.jquery.debug.js to check, and the option is never passed to the DragAndDropHandler.

I fixed the issue with this modification:

```
connectHandlers() {
        const {
          autoEscape,
          buttonLeft,
          closedIcon,
          dataFilter,
          dragAndDrop,
          keyboardSupport,
          onCanMove,
          onCanMoveTo,// MY FIX HERE
          onCreateLi,
          onDragMove,
          onDragStop,
          onGetStateFromStorage,
          onIsMoveHandle,
          onLoadFailed,
          onLoading,
          onSetStateFromStorage,
          openedIcon,
          openFolderDelay,
          rtl,
          saveState,
          showEmptyFolder,
          slide,
          tabIndex
        } = this.options;
```

AND

```
const dndHandler = new DragAndDropHandler({
          autoEscape,
          getNodeElement,
          getNodeElementForNode,
          getScrollLeft,
          getTree,
          onCanMove,
          onCanMoveTo, // MY FIX HERE
          onDragMove,
          onDragStop,
          onIsMoveHandle,
          openFolderDelay,
          openNode,
          refreshElements,
          slide,
          $treeElement,
          triggerEvent
        });
```

**Comment by mbraak** [comment]
I'll have a look.

**Comment by mbraak** [comment]
This is fixed in version 1.8.1
